25lessons/script.js

```diff
@@ -0,0 +1,103 @@
+//1
+let number1 = +prompt("Enter first number");
+let number2 = +prompt("Enter second number");
+function number(number1, number2) {
+  if (number1 < number2) {
+    return -1;
+  } else if (number1 > number2) {
+    return 1;
+  } else {
+    return 0;
+  }
+}
+alert(number(number1, number2));
+
+//2
+let factorialNumber = +prompt("Enter number please:");
+function factorialOfNumber(factorialNumber) {
+  let number = 1;
+  for (let i = 1; i <= x; i++) {
+    number *= i;
+  }
+  return number;
+}
+alert(`${factorialOfNumber(factorialNumber)}`);
+
+3;
+
+let numb1 = +prompt("Enter first number please:");
+let numb2 = +prompt("Enter second number please:");
+let numb3 = +prompt("Enter third number please:");
+function allNumberTogether(numb1, numb2, numb3) {
+  let numberSum = String(numb1) + String(numb2) + String(numb3);
+  return numberSum;
+}
+alert(allNumberTogether(numb1, numb2, numb3));
+
+4;
+let lenghRectangle = +prompt("Enter please a lengh rectangle");
+let heighRectangle = +prompt("Enter please a heigh rectangle");
+
+function getArea(lengh, heigh) {
+  let area;
+  if (lengh === 0 || heigh === 0) {
+    area = lengh * lengh + heigh * heigh;
+  } else {
+    area = lengh * heigh;
+  }
+  return area;
+}
+
+alert(`${getArea(lenghRectangle, heighRectangle)}`);
+
+5;
+
+let number = +prompt("Please enter your number:");
+function isNumberPerfect(number) {
+  let w = 1;
+  let i = 0;
+  while (number > w) {
+    if (number % w === 0) {
+      i = i + w;
+    }
+    w++;
+  }
+  if (number === i) {
+    return " Your number perfect!";
+  } else {
+    return "Soryy.... but no";
+  }
+}
+alert(`${isNumberPerfect(number)}`);
+
+6;
+
+7;
+
+let hours = +prompt(`Please enter hours`);
+let minutes = +prompt(`Please enter  minutes`);
+let seconds = +prompt(`Please enter  seconds`);
+function yourTime(hours = "00", minutes = "00", seconds = "00") {
```

Comment:
функція приймає числа, тому й дефолтні значення - числа
```suggestion
function yourTime(hours = 0, minutes = 0, seconds = 0) {
```